### DISCUSSION TITLE
metrics-all index query rate suddenly increases

### DISCUSSION THREAD

**Discussion by xlq20080808** [apache/skywalking#10970] [Q&A]
skywalking oap 9.3.0
At a certain moment, the query qps of the index metrics-all increases and it seems that it will recover at 00:00 .
According to the nginx access log, there is no user accesses skywalking-ui at this moment.
This problem will cause the es cluster cpu usage too high.
2023-06-09
<img width="1990" alt="image" src="https://github.com/apache/skywalking/assets/33999699/c30126d4-1096-4d3f-a3bd-3b27727010d2">
2023-06-17
<img width="1980" alt="image" src="https://github.com/apache/skywalking/assets/33999699/9b2e562c-2cb7-4f06-97f6-9d69150bb4f4">
2023-06-20
<img width="1973" alt="image" src="https://github.com/apache/skywalking/assets/33999699/aca63785-ffc6-4825-b4dd-a018ba6c3fa4">
What might be the cause of this problem, and how should I troubleshoot it? thanks！

**Comment by wu-sheng** [comment]
This is your system, you should be able to find which queries are those. Can't you?

**Comment by xlq20080808** [comment]
Thank you for your reply
We have located the cause of the problem and submitted an issue #10998 , the discussion is closed
